tfjs-node/src/run_tests.ts

```diff
@@ -122,15 +122,15 @@ const IGNORE_LIST: string[] = [
   // Node backend which uses TF 2.4.0 doesn't support explicit padding
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=1 d=1 p=explicit',
   // tslint:disable-next-line:max-line-length
-  'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit defualt dimRoundingMode',
+  'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit default dimRoundingMode',
   // tslint:disable-next-line:max-line-length
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit dimRoundingMode=floor',
   // tslint:disable-next-line:max-line-length
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit dimRoundingMode=round',
   // tslint:disable-next-line:max-line-length
   'pool test-tensorflow {} max x=[3,3,1] f=[3,3] s=3 d=1 p=explicit dimRoundingMode=ceil',
   // tslint:disable-next-line:max-line-length
-  'pool test-tensorflow {} avg x=[3,3,1] f=[3,3] s=3 d=1 p=explicit defualt dimRoundingMode',
+  'pool test-tensorflow {} avg x=[3,3,1] f=[3,3] s=3 d=1 p=explicit default dimRoundingMode',
```

Comment:
These test names may need to be updated in tfjs-core as well, or they might not be excluded in the tfjs-node tests.

Reply:
Hi, @mattsoulanille, @gbaned 

I apologize for the delayed response and I have updated test names in tfjs-core, please refer this PR [#8184](https://github.com/tensorflow/tfjs/pull/8184) and I appreciate your time and look forward to your feedback. Thank you.